_inc/client/general-settings/site-plan.jsx

```diff
@@ -48,7 +48,6 @@ export const SitePlan = React.createClass( {
 export default connect(
 	( state ) => {
 		return {
-			getSiteConnectionStatus: () => getSiteConnectionStatus( state ),
```

Comment:
Made a couple of changes. It should work fine now. Previously I was getting an error when, having the plans card open, I disconnected the site.